hedera-mirror-rest/controllers/contractController.js

```diff
@@ -775,6 +787,30 @@ const extractContractLogsByIdQuery = (filters, contractId) => {
     inValues[constants.filterKeys.TIMESTAMP],
     keyFullNames[constants.filterKeys.TIMESTAMP]
   );
+  updateQueryFiltersWithInValues(
```

Comment:
COuld simplify this section with something like
```suggestion
  Object.keys(keyFullNames).forEach((key) => {
    updateQueryFiltersWithInValues(
      params,
      conditions,
      inValues[key],
      keyFullNames[key]
    );
  });
```